docs/Buttons.md

```diff
@@ -299,6 +299,35 @@ const ListActions = () => (
 ## Record Buttons
 
 ### `<DeleteButton>`
+
+### `<DeleteWithConfirmButton>`
+
+Delete the selected element. To be used inside a <Toolbar> conponent.
+Overriding translateOptions prop allows to make custom message for the deletion.
+
+
+```jsx
+import * as React from 'react';
+import { Toolbar, SimpleForm } from '../form';
+
+translateOptions = {
+    id: 'custom identifier',
+    name: 'custom name'
+}
+
+const EditToolbar = props => (
+    <Toolbar {...props}>
+        <DeleteWithConfirmButton translateOptions={translateOptions} />
+    </Toolbar>
+);
+```
+
+| Prop         | Required | Type            | Default            | Description                         |
```

Comment:
Without an introduction, this table is misleading - it looks like the description of the `DeleteWithConfirmButton` props

Reply:
I was inspired by previous tables in the documentation, in order to describe the changes applied by my PR (concerning the two props).
What should this table describe ?